### ISSUE TITLE
textacy not installable with python 3.7

### ISSUE THREAD

**Issue by bollwyvl** [conda-forge/textacy-feedstock#16]
Issue:

Textacy is not currently installable with python 3.7.

Possible options: happy to help, but would like to know the maintainers' thoughts!
- [ ] remove the pin on this and pray (probably a bad idea)
- [ ] try to get textacy to work with newer ftfy (seems a bit out of scope)
- [ ] get a backported 4.x on python 3.7 build from @conda-forge/ftfy (seems reasonable)

Environment (<code>conda list</code>):
<details>

```
$ conda create -n textacy-test -c conda-forge python=3.7 textacy
Collecting package metadata: done
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - python=3.7
  - textacy -> ftfy[version='>=4.2,<5.0.0'] -> python=3.6

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
active environment : _conda_forge
active env location : ~/miniconda3/envs/_conda_forge
       shell level : 1
  user config file : ~/.condarc
populated config files : 
     conda version : 4.6.14
conda-build version : 3.17.8
    python version : 3.7.2.final.0
  base environment : ~/miniconda3  (writable)
      channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                     https://repo.anaconda.com/pkgs/main/noarch
                     https://repo.anaconda.com/pkgs/free/linux-64
                     https://repo.anaconda.com/pkgs/free/noarch
                     https://repo.anaconda.com/pkgs/r/linux-64
                     https://repo.anaconda.com/pkgs/r/noarch
     package cache : ~/miniconda3/pkgs
                     ~/.conda/pkgs
  envs directories : ~/miniconda3/envs
                     ~/.conda/envs
          platform : linux-64
        user-agent : conda/4.6.14 requests/2.21.0 CPython/3.7.2 Linux/5.0.9-200.fc29.x86_64 fedora/29 glibc/2.28
           UID:GID : 1000:1000
        netrc file : None
      offline mode : False

```
</details>

**Comment by bdewilde** [comment]
Hey @bollwyvl , thanks for the heads-up. fwiw I'm actually dropping the dependency on `ftfy` in the next release of textacy, since I was just exposing a single, much simpler entry point to the underlying functionality. It seems better to let users fix mangled unicode in whatever way they deem fit.

**Comment by bollwyvl** [comment]
Sounds great! In that case, I have no problem keeping up an older port
internally until the new release is available.

**Comment by bdewilde** [comment]
Hi again, I've merged in the PR for v0.7.0, which does not have the `ftfy` dependency. Hope this solves your issue!
